middleware/logger_test.go

```diff
@@ -78,3 +79,119 @@ func TestLoggerIPAddress(t *testing.T) {
 	h(c)
 	assert.Contains(t, ip, buf.String())
 }
+
+func TestLoggerHeaders(t *testing.T) {
```

Comment:
Make it just `TestLoggerTemplate` and use a template with all tags and assert on each.